doc/tutorials/Chapter_2/h_bus_range_swizzling.dart

```diff
@@ -0,0 +1,79 @@
+/// Copyright (C) 2023 Intel Corporation
```

Comment:
There doesn't seem to be any references to this file in the text. Perhaps it is worth mentioning it, as it is done for other files?